### ISSUE TITLE
jet black skin color on spawn

### ISSUE THREAD

**Issue by mhare323432** [ARF-SS13/coyote-bayou#1078]
-revision and game information
 Round ID: 839
BYOND Version: 514.1583
Server revision compiled on: 2022-11-03T04:45:11+00:00
Master commit: [f224cc35dc32c2e4f6d7e9895a8b5bda73c49f13](https://github.com/ARF-SS13/coyote-bayou/commit/f224cc35dc32c2e4f6d7e9895a8b5bda73c49f13)
TGS version: 4.17.1
DMAPI version: 5.3.0

Current Informational Settings:
Protect Authority Roles From Traitor: 0
Protect Assistant Role From Traitor: 0
Enforce Human Authority: 0
Allow Latejoin Antagonists: 1

-issue-
 When using my human character, he spawns with jet black skin color and funky looking eyes and mouth. I have had this some times before but sometimes it doesn't happen. I have no clue what is going on.
Images:

 what it looks like 
![image](https://user-images.githubusercontent.com/57967900/199882517-19f00b19-ab01-4976-8017-d5dbd4d674b4.png)

what it should look like
![image](https://user-images.githubusercontent.com/57967900/199882763-2996e973-99f5-46a1-9393-73e7bd77e39c.png)

**Comment by Tk420634** [comment]
Yeah, we've seen this a few times.  Not really sure what's causing it.  S'very strange...  We'll do what we can tho!

**Comment by GremlingSS** [comment]
Is this clientside or can other people can see this too? Very strange, message me on discord if you get the issue again <3
